README.md

```diff
@@ -1,113 +1,4 @@
-- [Exercise 11 - Deployment to SAP BTP, Cloud Foundry runtime](exercises/ex11/)
-    - [Exercise 11.1 - Create Space in Cloud Foundry](exercises/ex11#exercise-111---create-space-in-cloud-foundry)
-    - [Exercise 11.2 - Login to Cloud Foundry](exercises/ex11#exercise-112---login-to-cloud-foundry)
-    - [Exercise 11.3 - Set Organization and Space](exercises/ex11#exercise-113---set-organization-and-space)
-    - [Exercise 11.4 - Configure App Router for Redirect](exercises/ex11#exercise-114---configure-the-app-router-for-redirect)
-    - [Exercise 11.5 - Configure Your UI5 Application](exercises/ex11#exercise-115---configure-your-ui5-application)
-    - [Exercise 11.6 - Build Your Application](exercises/ex11#exercise-116---build-your-application)
-    - [Exercise 11.7 - Deploy Your Application](exercises/ex11#exercise-117---deploy-your-application)
-
-## Download and Installation
-Follow these [steps](https://github.com/SAP-samples/teched2020-DEV164/tree/code) to get the code or do the exercises step by step.
-
-## Known Issues
-No known issues.
-
-## Contributing
-The content of this repository is targeting the session *DEV164 - Developing Apps with SAPUI5* at SAP TechEd 2020. Only sesson speakers will make contributions to this repository. If you wanna enhance the content of this repository feel free to fork it.
-
-## How to obtain support
-Support for the content in this repository is available during the actual time of the online session for which this content has been designed. Otherwise, you may request support via the [Issues](../../issues) tab.
-
-## Stay in touch
-Stay in touch with the community by choosing your favourite channel.
-
-**UI5**:
-* Landing page: [OpenUI5 Website](https://openui5.org/)
-* Podcasts series: [UI5 Newscast](https://podcast.opensap.info/ui5-newscast/)
-* Blog posts: [UI5ers Buzz](https://blogs.sap.com/tag/ui5ers-buzz/)
-* Twitter: [@openui5](https://twitter.com/openui5)
-* Slack: [Get Access to OpenUI5 Slack Workspace](https://ui5-slack-invite.cfapps.eu10.hana.ondemand.com/)
-
-**SAP Open Source**:
-* Landing page: [Run better together with open source](https://developers.sap.com/open-source.html)
-* Podcasts series: *[The Open Source Way](https://podcast.opensap.info/open-source-way/2020/10/28/corona-warn-app-behind-the-scenes/)*
-* Blog posts: [open source on SAP Community](https://blogs.sap.com/tags/b2aac474-1581-4b1b-8932-de5f60b52558/)
-* Twitter: [@sapopensource](https://twitter.com/sapopensource)
-
-## License
-
-Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, version 2.0 except as noted otherwise in the [LICENSE](/LICENSES/Apache-2.0.txt) file.
+# Developing Apps with SAPUI5
+The workshop material has been moved to:
+[https://pages.github.tools.sap/ui5-training/ui5-beginners-tutorial](https://pages.github.tools.sap/ui5-training/ui5-beginners-tutorial)
```

Comment:
have sap externals access to github.tools.sap? Is this recommended by the TechEd team?